WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeSearchFragment.java

```diff
@@ -18,17 +19,15 @@
 /**
  * A fragment for display the results of a theme search
  */
-public class ThemeSearchFragment extends ThemeTabFragment implements SearchView.OnQueryTextListener, MenuItemCompat.OnActionExpandListener {
+public class ThemeSearchFragment extends ThemeBrowserFragment implements SearchView.OnQueryTextListener,
+        MenuItemCompat.OnActionExpandListener {
     public static final String TAG = ThemeSearchFragment.class.getName();
     private static final String BUNDLE_LAST_SEARCH = "BUNDLE_LAST_SEARCH";
+    public static final int SEARCH_VIEW_MAX_WIDTH = 10000;
```

Comment:
Huge width. Does it not format correctly on tablets without setting it to a large enough number?